### PR TITLE
more changes from feedback

### PR DESCRIPTION
**Pull request by harleenkohli**
changes in this PR:

* API NON-BREAKING: Add a new value to `result.kind`: `informational`.
* API NON-BREAKING: add `address.kind` values `function` and `page`.
* API NON-BREAKING: `run.columnKind` has no default value.
* API NON-BREAKING: In the `reportingDescriptorRelationship` object, add a property `description` of type message, optional.
* API NON-BREAKING: In the `locationRelationship` object, add a property `description` of type message, optional.

### CODE REVIEW

File: src/ReleaseHistory.md [hunk 1]
```diff
@@ -358,3 +358,8 @@
 * API BREAKING: `threadFlowLocation.executionOrder` can be 0 and defaults to -1, so minimum: -1, default: -1
 * API BREAKING: Rename artifact role `traceFile` to `tracedFile`.
 * API NON-BREAKING: Add artifact role `debugOutputFile`.
+* API NON-BREAKING: Add a new value to `result.kind`: `informational`.
+* API NON-BREAKING: add `address.kind`�values `function`�and `page`.
+* API NON-BREAKING: `run.columnKind`�has no default value.
```
Comment (michaelcfanning) [inline src/ReleaseHistory.md]: >� [](start = 36, length = 1)

do you know what these unicode characters are in the doc? a strange non-breaking space or something? try opening in notepad and stripping them.

Comment (michaelcfanning) [reply src/ReleaseHistory.md]: then merge!

---
In reply to: [278141596](https://github.com/Microsoft/sarif-sdk/pull/1455#discussion_r278141596) [](ancestors = 278141596)

File: src/ReleaseHistory.md [hunk 2]
```diff
@@ -358,3 +358,8 @@
...
+* API NON-BREAKING: In the `reportingDescriptorRelationship`�object, add a property `description`�of type message, optional.
```
Comment (michaelcfanning) [inline src/ReleaseHistory.md]: >message [](start = 106, length = 7)

encapsulate `message` here and in the other line in backticks

File: src/ReleaseHistory.md (hunk 1 above)
Comment (harleenkohli) [reply src/ReleaseHistory.md]: woah! this looks ugly! let me remove it :)

---
In reply to: [278141687](https://github.com/Microsoft/sarif-sdk/pull/1455#discussion_r278141687) [](ancestors = 278141687,278141596)

**Pull request merged**